harvester_e2e_tests/integration/test_networks.py

```diff
@@ -0,0 +1,840 @@
+
+        # Update VM spec
+        code, data = api_client.vms.update(unique_name, spec)
+
+        code, data = api_client.vms.restart(unique_name)
+
+        # Check VM start in running state
+        check_vm_running(api_client, unique_name, wait_timeout)
+
+        # Check until VM ip address exists
+        check_vm_ip_exists(api_client, unique_name, wait_timeout)
+
+        # get data from running VM and transfer to spec
+        code, data = api_client.vms.get(unique_name)
+        spec = spec.from_dict(data)
+
+        # Delete external vlan from VM spec
+        spec.delete_vlan_network(spec, "default/" + vlan_network['id'])
+
+        code, data = api_client.vms.update(unique_name, spec)
+
+        code, data = api_client.vms.restart(unique_name)
+
+        # Check VM start in running state
+        endtime = datetime.now() + timedelta(seconds=wait_timeout)
+
+        while endtime > datetime.now():
+            code, data = api_client.vms.get(unique_name)
+            vm_fields = data['metadata']['fields']
+
+            assert 200 == code, (code, data)
+            if vm_fields[2] == 'Running':
+                break
+            sleep(5)
+        else:
+            raise AssertionError(
+                f"Failed to create VM {unique_name} in Running status, exceed 10 minutes\n"
+                f"Still got {code} with {data}"
+            )
+
+        # Check until VM ip address exists
+        endtime = datetime.now() + timedelta(seconds=wait_timeout)
+
+        while endtime > datetime.now():
+            code, data = api_client.vms.vm_instance(unique_name)
+            if 'ipAddress' in data['status']['interfaces'][0]:
+                break
+
+            sleep(5)
+        else:
+            raise AssertionError(
+                f"Failed to get VM {unique_name} IP address, exceed 10 minutes timed out\n"
+                f"Still got {code} with {data}"
+            )
+
+        endtime = datetime.now() + timedelta(seconds=wait_timeout)
+        ip_addresses = []
+
+        while endtime > datetime.now():
+            code, data = api_client.vms.vm_instance(unique_name)
+            if 'interfaces' in data['status']:
+                interfaces_data = data['status']['interfaces']
+                ip_addresses = []
+
+                if 'ipAddress' in data['status']['interfaces'][0]:
+                    ip_addresses.append(interfaces_data[0]['ipAddress'])
+
+                    if '10.52' in ip_addresses[0]:
+                        break
+                sleep(5)
+        else:
+            raise AssertionError(
+                f"Failed to get VM {unique_name} IP address, exceed 10 minutes timed out\n"
+                f"Still got {code} with {data}"
+            )
+
+        # Ping management ip address
+        mgmt_ip = ip_addresses[0]
+
+        ping_command = "ping -c 50 {0}".format(mgmt_ip)
+
+        _stdout, _stderr = self.ssh_client(
+            client, client_ip, "rancher", "p@ssword", ping_command, wait_timeout)
```

Comment:
It pretty weird, you already constant `node_user` and `node_password`, and still hard coded them in here?

Reply:
@TachunLin Please help change to constant and also check if there are similar cases too. Thanks!

Reply:
Appreciated for the review. 
I will update the hard coded at line 804 to the existing constant `node_user` and `node_password`. 
And also check no hard code added as the function parameter in test cases.